Mathlib/CategoryTheory/ConcreteCategory/Basic.lean

```diff
@@ -187,6 +187,13 @@ theorem ConcreteCategory.bijective_of_isIso {X Y : C} (f : X ⟶ Y) [IsIso f] :
   infer_instance
 #align category_theory.concrete_category.bijective_of_is_iso CategoryTheory.ConcreteCategory.bijective_of_isIso
 
+/-- If the forgetful functor of a concrete category reflects isomorphisms, being an isomorphism
+is equivalent to being bijective. -/
+theorem ConcreteCategory.isIso_iff_bijective_of_reflectsIso [ReflectsIsomorphisms (forget C)]
```

Comment:
As the parameter `ReflectsIsomorphisms` the obviously expected typeclass, I think we may shorten the name:
```suggestion
theorem ConcreteCategory.isIso_iff_bijective [ReflectsIsomorphisms (forget C)]
```